### PR TITLE
chores: update chaos mesh adopters

### PR DESCRIPTION
**Pull request by milasuperstar**
Signed-off-by: Mila Wu <wuzaixuan@pingcap.com>

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

* PR to update `chaos-mesh/website`/`chaos-mesh/website-zh`:
* Need to update Chaos Dashboard component, related issue:
* Need to cheery-pick to the release branch

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

**Comment by milasuperstar** [comment]
@dcalvin PTAL

**Review by fewdan — APPROVED** [review]
LGTM

File: ADOPTERS.md [hunk 1]
```diff
@@ -1,22 +1,45 @@
...
+        - Fault isolation, such as simulating pod abnormality, and checking whether the system can automatically isolate fault instances;
+        Service degradation, such as simulating a downstream recommended service failure through network failure, and verifying whether the local cache is effective;
+        - Verifying if the alarm works, for example, purposefully burning the CPU to 90%, and checking whether the alarm is timely issued in time.
 - [Vald](https://vald.vdaas.org/)
 - [WeBank](https://www.webank.com/)
 - [Xpeng](https://en.xiaopeng.com/)
+    - Xpeng Motors is China's leading smart electric vehicle designer and manufacturer, as well as a technology company integrating cutting-edge Internet and AI innovation. They use Chaos Mesh in the following scenarios:
+        - Rolling updates of microservices and lossless verification of traffic;
+        - Microservices, multi-registries, multi-party synchronization, and traffic lossless verification;
+        - mqtt cluster two-way subscription verification;
```
Comment (Gallardot) [inline ADOPTERS.md:42]:
```suggestion
        - MQTT cluster two-way subscription verification;
```

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**